open-api-spec.yaml

```diff
@@ -793,6 +900,68 @@ components:
           $ref: '#/components/schemas/amount'
         sendAmount:
           $ref: '#/components/schemas/amount'
+        description:
+          type: string
+          description: Human readable description of the outgoing payment that will be visible to the account holder and shared with the receiver.
+          example: APlusVideo subscription
+        externalRef:
+          type: string
+          description: A reference that can be used by external systems to reconcile this payment with their systems. E.g. An invoice number. (Optional)
+          example: 'customer: 847458475'
+    quote:
+      title: Quote
+      description: 'A **quote** resource represents a payment that will be, is currently being, or has previously been, sent from the account.'
+      type: object
+      examples:
+        - id: 'https://openpayments.guide/alice/quotes/ab03296b-0c8b-4776-b94e-7ee27d868d4d'
+          accountId: 'https://openpayments.guide/alice/'
+          receivingAccount: 'https://openpayments.guide/aplusvideo/'
+          receiveAmount:
+            amount: 2500
+            assetCode: USD
+            assetScale: 2
+          sendAmount:
+            amount: 2500
+            assetCode: USD
+            assetScale: 2
+          fundsAvailable: true
+          expiresAt: '2022-04-12T23:20:50.52Z'
+        - id: 'https://openpayments.guide/alice/quotes/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+          accountId: 'https://openpayments.guide/alice/'
+          receivingPayment: 'https://openpayments.guide/aplusvideo/incoming-payments/45d495ad-b763-4882-88d7-aa14d261686e'
+          sendAmount:
+            amount: 2500
+            assetCode: USD
+            assetScale: 2
+          fundsAvailable: true
+          expiresAt: '2022-04-12T23:20:50.52Z'
+      properties:
+        id:
+          type: string
+          format: uri
+          description: The URL identifying the quote.
+          example: 'https://openpayments.guide/alice/outgoing-payments/8c68d3cc-0a0f-4216-98b4-4fa44a6c88cf'
+          readOnly: true
+        accountId:
+          type: string
+          format: uri
+          description: The URL of the account from which this quote's payment would be sent.
+          example: 'https://openpayments.guide/alice/'
+          readOnly: true
+        receivingAccount:
+          type: string
+          format: uri
+          description: The URL of the account that would be paid.
+          example: 'https://openpayments.guide/aplusvideo/'
+        receivingPayment:
+          type: string
+          format: uri
+          description: The URL of the incoming payment that would be paid.
+          example: 'https://openpayments.guide/aplusvideo/incoming-payments/45d495ad-b763-4882-88d7-aa14d261686e'
+        receiveAmount:
+          $ref: '#/components/schemas/amount'
+        sendAmount:
+          $ref: '#/components/schemas/amount'
```

Comment:
The quote should have either:
- [`paymentType`](https://github.com/interledgerjs/interledgerjs/tree/master/packages/pay#quote) to indicate fixed send vs fixed receive
- `maxSendAmount` (with `receiveAmount`) and `maxReceiveAmount` (with `sendAmount`)

Reply:
added `amountType`
https://github.com/interledger/open-payments/pull/97/commits/a8f520302e3798f61968c57dc085959865ebe3d6